lua/bufferline/types.lua

```diff
@@ -4,15 +4,15 @@
 ---@field logging boolean
 
 ---@class bufferline.GroupOptions
----@field toggle_hidden_on_enter boolean re-open hidden groups on bufenter
+---@field toggle_hidden_on_enter? boolean re-open hidden groups on bufenter
 
 ---@class bufferline.GroupOpts
----@field options bufferline.GroupOptions
+---@field options? bufferline.GroupOptions
 ---@field items bufferline.Group[]
 
 ---@class bufferline.Indicator
----@field style "underline" | "icon" | "none"
----@field icon string?
+---@field style? "underline" | "icon" | "none"
+---@field icon? string?
```

Comment:
If adding `?` after the field/param name, then maybe it might be a good idea to also remove the `?` after the type to make it consistent.

```suggestion
---@field icon? string
```

I think there are also others like this in this file:
lines 15, 37, 42, 50`*`, 181`*`

`*` = for those lines I am not completely sure